research/blanks/loss.py

```diff
@@ -143,11 +141,16 @@ def calculate_llm_loss(
                         assert not input_tokens[nth_blank_mask == 1].eq(blank_id).any()
                     else:
                         assert input_tokens[nth_blank_mask == 1].eq(blank_id).all()
+
                     nth_blank_loss = (nth_blank_mask.reshape(-1) * mask_loss).sum()
+
```

Comment:
this likely calls for a test for correctness

Reply:
Not really the point of this PR, but yeah, we try to have as many tests and asserts as possible in blanx.